cc/main.cc

```diff
@@ -145,6 +151,17 @@ std::string GetOutputName(absl::Time now, size_t i) {
   return absl::StrCat(timestamp, "-", hostname, "-", i);
 }
 
+std::string GetRowPrefix(absl::Time now, int id) {
+  std::ostringstream out;
+  absl::TimeZone utc = absl::UTCTimeZone();
+  out << absl::FormatTime("%Y%m%d-%H%M%S", now, utc)
```

Comment:
Can you use absl::StrFormat here? I find it easier to read than iomanip.

something like:

```
absl::StrFormat("%s-w%03d-", absl::FormatTime("%Y%m%d-%H%M%S", now, utc), id)
```

We do have a bunch of ostringstring in the C++ codebase because that was written before absl::StrFormat was released. I'll file an issue to clean those up.

Reply:
Actually, we might need to update our version of absl before you can use absl::StrFormat. Let me do that now.

Reply:
No problem. Will move to `absl::StrFormat`.